src/app/ReadHandler.h

```diff
@@ -290,13 +290,19 @@ class ReadHandler : public Messaging::ExchangeDelegate
     bool IsFromSubscriber(Messaging::ExchangeContext & apExchangeContext) const;
 
     bool IsIdle() const { return mState == HandlerState::Idle; }
-    bool IsReportable() const
+
+    /// @brief Returns whether the ReadHandler is in a state where it can immediately send a report. The main conditions for this
```

Comment:
The part of this comment between "the main conditions" and "is dirty" just restates the code and can probably be removed.  The sentences before/after that are useful.

The @return just restates the sentence one line above, and is not that useful.